cmake/os/AIX.cmake

```diff
@@ -14,20 +14,24 @@
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA 
 
 
+# This will never work with AIX, so switching off per default
+option(WITH_UNIT_TESTS "WITH_UNIT_TESTS" OFF)
+option(WITH_JEMALLOC "WITH_JEMALLOC" NO)
```

Comment:
Why? If jemalloc is not available, isn't it just skipped automatically by default?